app/Tasks/Status.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Tasks;
+
+use App\Builder\Docker;
+use App\Builder\DockerCompose;
+use App\Builder\Escaped;
+
+class Status extends Task
+{
+    /** @var bool $quite */
+    protected $quite = false;
+
+    public function run(...$args): int
+    {
+        $start = Start::make($this->command);
+
+        $chain = [
+            [$start, 'handleNetwork'],
+            [$this, 'displayServices'],
+        ];
+
+        return $this->runCallables($chain);
+    }
+
+    public function quite(bool $quite) : self
+    {
+        $this->quite = $quite;
+
+        return $this;
+    }
+
+    public function displayServices() : int
+    {
+        $lines = $this->getOutputLines(DockerCompose::make('ps', '--services'));
+
+        $services = collect($lines)
+            ->sort()
+            ->map(function ($service) {
+                $info = [];
+                $isRunning = $this->isRunning($service, $info);
+
+                return [
+                    $service,
+                    $isRunning
+                        ? '<fg=white;bg=green;options=bold>Running</>'
+                        : '<fg=white;bg=red;options=bold>Not running</>',
+                    $info['ports'],
+                    $info['state'],
+                ];
+            })
+            ->toArray();
+
+        $this->command->table([
+            'Service',
+            'Status',
+            'Ports',
+            'State',
+        ], $services);
+
+        return 0;
+    }
+
+    public function isRunning(string $service, array &$info) : bool
+    {
+        $lines = $this->getOutputLines(DockerCompose::make('ps', $service));
```

Comment:
`-q` for simplifying `isRunning` check:

Not running:

![image](https://user-images.githubusercontent.com/347400/71531323-f3fb6600-28cc-11ea-92ad-64e9ccdba9ea.png)

Running:

![image](https://user-images.githubusercontent.com/347400/71531339-06759f80-28cd-11ea-9cd4-fd511314965c.png)

Reply:
also use ID for filtering the container later on `docker ps`